src/osvr/Util/LogRegistry.cpp

```diff
@@ -187,7 +205,7 @@ namespace util {
             // File sink - rotates daily
             std::string logDir;
             try {
-                size_t q_size = 1048576; // queue size must be power of 2
+                size_t q_size = 65536; // queue size must be power of 2
```

Comment:
We could go _completely_ synchronous if you want. But if people use logging more frequently (e.g., as I do at higher verbosity levels), then it may cause lag.

The queue does get flushed on exit now. (Though I think we need to explicitly call `flush()` in any signal handlers we've established because they bypass the destructors.) Further, it messages of a specified log level and above should trigger an immediate flush.